lib/ttfunk/table/name.rb

```diff
@@ -23,6 +23,7 @@ def strip_extended
         end
       end
 
+      attr_reader :entries
```

Comment:
It seems it's possible to get all the data from `strings` without having to keep the entries in the table instance.

Reply:
The problem is that you lose the original order in `strings`. The original order is really only needed for verification in the tests, but it seemed easier to expose the `entries` array in the table than perform relatively complicated parsing from the raw file in the test. I can do that though, if you prefer to keep the `entries` array private.

Reply:
Yes, I see… I'd prefer to keep it private but don't want to make the test too complex. Let's keep it like this for now.